advanced/Scripts/piholeDebug.sh

```diff
@@ -652,7 +652,7 @@ check_required_ports() {
   # Sort the addresses and remove duplicates
   while IFS= read -r line; do
       ports_in_use+=( "$line" )
-  done < <( lsof -i -P -n | awk -F' ' '/LISTEN/ {print $9, $1}' | sort -n | uniq | cut -d':' -f2 )
+  done < <( lsof -i -P -n | awk -F' ' '/LISTEN/ {print $1, $9}' | sort -n | tr -d '[*]\r' | uniq | awk '{print $2, $1}' )
```

Comment:
`lsof -iTCP -sTCP:LISTEN -P -n +c 10`

List open files of internet TCP type, in the listen state, convert port names to numbers and don't look up IP address to name resolution. Also set `command` field to 10 chars so we don't lop off the last part of FTL.

Also, do we want `uniq` on this? I think it may be better to capture the IPv4 and IPv6 TYPE, so just sort and include that field in the output?

Reply:
You're amazing.  I think we should do all that you have suggested.

Reply:
OK I implemented it and it now looks like this:
```
*** [ DIAGNOSING ]: Ports in use
*:111 systemd (IPv6)
*:111 systemd (IPv4)
*:4711 openvpn (IPv4)
*:22 sshd (IPv4)
*:22 sshd (IPv6)
*:80 lighttpd (IPv4)
*:53 pihole-FTL (IPv4)
*:53 pihole-FTL (IPv6)
127.0.0.1:4711 pihole-FTL (IPv4)
[::1]:4711 pihole-FTL (IPv6)
```